pkg/metrics/metrics.go

```diff
@@ -229,6 +248,26 @@ func (hm *hcoMetrics) IsHCOMetricHyperConvergedExists() (bool, error) {
 	return val == HyperConvergedExists, nil
 }
 
+// SetHCOMetricSystemHealthStatusHealthy sets the gauge to 0 (healthy)
+func (hm *hcoMetrics) SetHCOMetricSystemHealthStatusHealthy() error {
+	return hm.SetMetric(HCOMetricSystemHealthStatus, nil, SystemHealthStatusHealthy)
+}
+
+// SetHCOMetricSystemHealthStatusWarning sets the gauge to 1 (warning)
+func (hm *hcoMetrics) SetHCOMetricSystemHealthStatusWarning() error {
+	return hm.SetMetric(HCOMetricSystemHealthStatus, nil, SystemHealthStatusWarning)
+}
+
+// SetHCOMetricSystemHealthStatusError sets the gauge to 2 (error)
+func (hm *hcoMetrics) SetHCOMetricSystemHealthStatusError() error {
+	return hm.SetMetric(HCOMetricSystemHealthStatus, nil, SystemHealthStatusError)
+}
```

Comment:
Can we have one function that will get the status as a parameter? Not sure what is the advantage to write three functions. The reason I'm rising it is the `updateSystemHealthStatusMetric` function, that can be much simpler - just pass the current status to a single function.

Reply:
Sure, makes sense. 
Should we then keep `updateSystemHealthStatusMetric()` in the controller, or to have all of this logic in the metrics package?

Reply:
Done